### PR TITLE
Added missing Highlights information to 2.3.2 Open Source release notes

### PR DESCRIPTION
**Pull request by jfrontain**
## Purpose of this pull request

This pull request (PR)  will add the merchant tools-related Highlights information to the Magento Open Source 2.3.2 Release Notes

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.3/release-notes/ReleaseNotes2.3.2OpenSource.html

- ...
- ...

## Links to Magento source code

- ...
- ...

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=4835) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=4835) before we can accept your contribution.<br/><hr/>**Jeanne Frontain** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

File: guides/v2.3/release-notes/ReleaseNotes2.3.2OpenSource.md [hunk 1]
```diff
@@ -58,6 +58,46 @@ Magento now performs the following tasks as **asynchronous background processes*
 
 * Data export. Previously, connection timeouts occurred during export of large data sets (for example, the export of 200,000 products). See [Export](https://docs.magento.com/m2/b2b/user_guide/system/data-export.html) for more information. 
  <!--- MC-5953-->
+### Inventory Management enhancements
```
Comment (keharper) [inline guides/v2.3/release-notes/ReleaseNotes2.3.2OpenSource.md]: Add a blank line above the heading.

File: guides/v2.3/release-notes/ReleaseNotes2.3.2OpenSource.md [hunk 2]
```diff
@@ -58,6 +58,46 @@ Magento now performs the following tasks as **asynchronous background processes*
...
+  * Multiple product save
+  * Bulk inventory transfer
+
+* **New endpoint** for Bulk Partial Stock Transfer to bulk transfer a custom product quantity between sources. See [Inventory Bulk Actions](https://devdocs.magento.com/guides/v2.3/rest/modules/inventory/bulk-inventory.html).
+
+* Fixes to multiple  bugs. See [Inventory Management release notes](https://devdocs.magento.com/guides/v2.3/inventory/release-notes.html).
+
+### GraphQL
+
+GraphQL performance improvements include these enhancements:
```
Comment (keharper) [inline guides/v2.3/release-notes/ReleaseNotes2.3.2OpenSource.md]:
```suggestion
GraphQL enhancements include:
```

File: guides/v2.3/release-notes/ReleaseNotes2.3.2OpenSource.md [hunk 3]
```diff
@@ -58,6 +58,46 @@ Magento now performs the following tasks as **asynchronous background processes*
...
+
+* **GraphQL caching**. GraphQL can now cache the `category`, `cmsBlocks`, `cmsPage`, `products`, and `urlResolver` queries, which improves response times.  To enable this feature, send these queries using HTTP GET. You can send all other queries with HTTP GET, but they are not cached.
```
Comment (keharper) [inline guides/v2.3/release-notes/ReleaseNotes2.3.2OpenSource.md]: Make this the second bullet.

File: guides/v2.3/release-notes/ReleaseNotes2.3.2OpenSource.md (hunk 1 above)
Comment (jfrontain) [reply guides/v2.3/release-notes/ReleaseNotes2.3.2OpenSource.md]: got it -- thanks

**Comment by jfrontain** [comment]
running tests

**Comment by jfrontain** [comment]
running tests

**Pull request merged**